src/loaders/Loader.js

```diff
@@ -1,21 +1,21 @@
 import { DefaultLoadingManager } from './LoadingManager.js';
 
-function Loader( manager ) {
+class Loader {
```

Comment:
You can't convert the `Loader` class. I've explained this multiple times. Example loaders inherit from `Loader` and thus you can't do the class conversion now. Otherwise you have to migrate all example loaders which is currently not possible.

Reply:
@Mugen87 Don't worry, we'll deal with this.

Reply:
May I ask what's the plan? 😇

Reply:
I guess https://github.com/mrdoob/three.js/pull/19934#issuecomment-664241764?

Reply:
so... I know that, that's why this PR is named "Draft".

if the loaders change too much between here and there I can remake this PR.

example scripts will probably be part of the change.